platform/darwin/src/MGLBaseStyleLayer.h

```diff
@@ -14,4 +14,11 @@
  */
 @property (nonatomic, assign) float minimumZoomLevel;
 
+@property (nonatomic) NSPredicate *predicate;
+
+/**
+ Updates the layer’s layout and paint properties.
+ */
+- (void)update;
```

Comment:
We made this method private in #6018. Any reason you’re reintroducing this method in the public interface?